test/e2e/framework/ingress_utils.go

```diff
@@ -761,9 +761,20 @@ func (cont *GCEIngressController) deleteStaticIPs() error {
 func gcloudComputeResourceList(resource, regex, project string, out interface{}) {
 	// gcloud prints a message to stderr if it has an available update
 	// so we only look at stdout.
+	var regexFlag string
```

Comment:
Usually it's better to add a new commit on each review round, so previous comments are not lost and also it's easier to verify if comments have been addressed.
However, it's ok to squash for simple PRs.